src/main/java/mate/academy/hibernate/relations/dao/impl/ActorDaoImpl.java

```diff
@@ -12,11 +16,38 @@ public ActorDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Actor add(Actor actor) {
-        return null;
+        SessionFactory sessionFactory = HibernateUtil.getSessionFactory();
+        Session session = sessionFactory.openSession();
```

Comment:
Instead of creating a new `SessionFactory` instance using `HibernateUtil.getSessionFactory()`, use the `SessionFactory` instance provided by the `AbstractDao` class (`this.factory`). This ensures consistency and avoids potential issues with multiple `SessionFactory` instances.